pkg/actors/actors_test.go

```diff
@@ -284,6 +284,92 @@ func TestCreateReminder(t *testing.T) {
 	assert.Nil(t, err)
 }
 
+func TestCreateTimer(t *testing.T) {
+	testActorsRuntime := newTestActorsRuntime()
+	actorType, actorID := getTestActorTypeAndID()
+	ctx := context.Background()
+	requestTests := []*CreateTimerRequest{
+		{
+			Name:      "timer1",
+			ActorID:   actorID,
+			ActorType: actorType,
+			DueTime:   "5s",
+			Period:    "2s",
+			Data:      nil,
+		},
+		{
+			Name:      "timer2",
+			ActorID:   actorID,
+			ActorType: actorType,
+			DueTime:   "5s",
+			Period:    "",
+			Data:      nil,
+		},
+		{
+			Name:      "timer3",
+			ActorID:   actorID,
+			ActorType: actorType,
+			DueTime:   "",
+			Period:    "2s",
+			Data:      nil,
+		},
+		{
+			Name:      "timer4",
+			ActorID:   actorID,
+			ActorType: actorType,
+			DueTime:   "0s",
+			Period:    "2s",
+			Data:      nil,
+		},
+		{
+			Name:      "timer4",
+			ActorID:   actorID,
+			ActorType: actorType,
+			DueTime:   "0",
+			Period:    "2s",
+			Data:      nil,
+		},
+	}
+	requestIllegalTests := []*CreateTimerRequest{
+		{
+			Name:      "timer5",
+			ActorID:   actorID,
+			ActorType: actorType,
+			DueTime:   "wehows",
+			Period:    "2s",
+			Data:      nil,
+		},
+	}
+	// register actor_type & actor_id
+	fakeCallAndActivateActor(testActorsRuntime, actorType, actorID)
+	for _, test := range requestIllegalTests {
+		err := testActorsRuntime.CreateTimer(ctx, test)
+		assert.NotNil(t, err)
+	}
+	for _, test := range requestTests {
+		err := testActorsRuntime.CreateTimer(ctx, test)
+		assert.Nil(t, err)
+	}
+}
+
+func TestStartTimer(t *testing.T) {
+	testActorsRuntime := newTestActorsRuntime()
+	actorType, actorID := getTestActorTypeAndID()
+	ctx := context.Background()
+	// register actor_type & actor_id
+	fakeCallAndActivateActor(testActorsRuntime, actorType, actorID)
+	stop := make(chan bool, 1)
+	go testActorsRuntime.startTimer(ctx, stop,
+		&CreateTimerRequest{
+			Name:      "timer1",
+			ActorID:   actorID,
+			ActorType: actorType,
+			DueTime:   "2s",
+			Period:    "1s",
+			Data:      nil,
+		})
+	time.Sleep(3 * time.Second)
```

Comment:
Please, modify the test so it does not depend on sleep and add validation.

/cc @pkedy